src/components/deviceMap/deviceMap.js

```diff
@@ -186,7 +186,8 @@ class DeviceMap extends Component {
 
 const mapStateToProps = state => {
   return {
-    showSpinner : state.indicatorReducer.map,
+    showSpinner : state.indicatorReducer.kpi,
```

Comment:
There are several places in this PR with spaces before the `:`. I'm not sure how they keep showing up. Are they being added automatically perhaps? Please do a sweep and remove them all.

Reply:
Sure.